testlib/src/test/resources/formatter/properties/valid_xml_profiles.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<profiles version="12">
+	<profile kind="First" name="Used Profile" version="SomeVersion">
+		<setting id="valid_xml_profiles.xml.string" value="string" />
+		<setting id="valid_xml_profiles.xml.true" value="Gets overridden by next line" />
+		<setting id="valid_xml_profiles.xml.true" value="true" />
+		<setting id="valid_xml_profiles.xml.42" value="42" />
+		<setting id="valid_xml_profiles.xml.null"/>
+		<setting id="common" value="valid_xml_profiles.xml" />
+	</profile>
+	<profile kind="Second" name="Skipped Profile" version="AnotherVersion">
+		<setting id="valid_xml_profiles.xml.string" value="Second profile should be skipped" />
+		<setting id="valid_xml_profiles.xml.true" value="Second profile should be skipped" />
+		<setting id="valid_xml_profiles.xml.42" value="Second profile should be skipped" />
+		<setting id="valid_xml_profiles.xml.null" value="Second profile should be skipped" />
+		<setting id="common" value="Second profile should be skipped" />
+	</profile>
+</profiles>
```

Comment:
Sorry,  just remembered that you stated explicitly in your documentation that only a [single profile shall be exported](https://github.com/diffplug/spotless/blob/master/ECLIPSE_SCREENSHOTS.md). Hence the usage of multiple profiles was in the past considered erroneous, and therefore a warning appeared.
For Eclipse-JDT, the order can not be configured. 
For the Formatter steps (and this is the only use case for this class), loading of multiple profiles is purposeless.
An extension of the configurability to select a profile can be considered unnecessarily complex.

So you are right, the usage of multiple profiles could lead to a failure.
But I should have never reduced the log-level to QUIET in the first place.

So I assume that you prefer a failure with a "informative" message how the user can fix the problem. I will implement the change correspondingly.
:+1: